ci/build_wheel.sh

```diff
@@ -5,23 +5,23 @@ set -euo pipefail
 
 package_name=$1
 package_dir=$2
-underscore_package_name=$(echo "${package_name}" | tr "-" "_")
 
 # The set of shared libraries that should be packaged differs by project.
 #
 # Capturing that here in argument-parsing to allow this build_wheel.sh
 # script to be re-used by all wheel builds in the project.
-case "${package_dir}" in
-  python/pylibwholegraph)
-    EXCLUDE_ARGS=(
-        --exclude libcuda.so.1
-        --exclude libnvidia-ml.so.1
+#
+EXCLUDE_ARGS=(
+    --exclude libcuda.so.1
+    --exclude libnvidia-ml.so.1
+    --exclude librapids_logger.so
+    )
+
+if [[ "${package_name}" == "pylibwholegraph" ]]; then
```

Comment:
```suggestion
if [[ "${package_name}" != "libwholegraph" ]]; then
```

I think it'd be a little safer to change this condition from "if is exactly `pylibwholegraph`" to "if is anything other than `libwholegraph`", so avoid accidentally vendoring `libwholegraph.so` in other wheels in this project, as the project evolves.

For example, did you double-checked that `libwholegraph.so` isn't now getting vendored by `auditwheel repair` in `cugraph-pyg` / `cugraph-dgl` builds, given the recently-merged changes in https://github.com/rapidsai/cugraph-gnn/pull/172 ?

Reply:
It's not getting vendored in right now, but I don't know if I'm up-to-date against the default branch.  And this seems like a sensible precaution no matter what!